ted_adapter/test_adapter.py

```diff
@@ -119,6 +158,12 @@ def main():
     :return: None
     """
 
+    if not driver_exists():
+        path = str(adapter_conf.DRIVER_PATH)
+        print(f"Driver does not exist in the following location: {path}")
```

Comment:
should be in driver_exists()
message should be like: Driver was not found: <full path to file>
Install the driver and run tests again. or see instructions here: <link>

Reply:
Why is it better to put to this function? This function is very simple and only for check. I think that it should not write any messages. It can be in separate function if you wish...